locale/en/blog/uncategorized/10-lts-to-12-lts.md

```diff
@@ -0,0 +1,4333 @@
+---
+date: 2019-10-20T12:00:00.000Z
+category: Uncategorized
+title: The Difference Between Node.js 10 LTS and Node.js 12 LTS
+slug: the-difference-between-node-js-10-lts-and-node-js-12-lts
+layout: blog-post.hbs
+author: Tierney Cyren
+---
+
+With the transition of Node.js 12 from LTS into Current, a suite of new features have been added to Node.js!
+
+In each release's changelog we capture notable changes – but going through over two dozen individual release notes to find those can be a bit daunting.
+
+As such, I've pulled together **all release notes** from every release from Node.js 11.0.0 to 12.12.0 and included an even higher level "Best of the Best: Most Notable Changes" section that allows you to see what some of the biggest changes are between Node.js v10.x LTS at launch and Node.js v12.x LTS at launch.
+
+## Best of the Best: Most Notable Changes
+
+This section is a highlight of the most notable changes, as selected from all "[Notable Changes](#notable-changess)" from every release since Node.js v10 went LTS.
```

Comment:
```suggestion
This section is a highlight of the most notable changes, as selected from all "[Notable Changes](#notable-changes)" from every release since Node.js v10 went LTS.
```